### ISSUE TITLE
GitHub to Jira POC

### ISSUE THREAD

**Issue by kaylareopelle** [newrelic/newrelic-ruby-agent#1285]
Our team is participating in a POC experiment to evaluate the ability to sync issues from our GitHub project to New Relic's new Jira instance. The intention is for developers to continue their work in GitHub, but have the important information synced on issues between both platforms to give product managers and other users of Jira the ability to reference the issues from GitHub.

## Tasks 
- [x] Mimic the conversation from an existing ticket on the newrelic-ruby-agent repo on the GitHub Jira Sandbox, going between my personal and New Relic GitHub accounts. Evaluate the differences/missing content (if anything) and share workflow notes with the team
  - [x] Completed on bug
  - [x] Completed on feature-request
  - [x] Completed on story 
- [x] Create story.md issue template to represent Story issue type in Jira
- [x] Change `feature` issue template to `feature-request` 
- [x] Update existing tickets labeled `feature` into either unlabeled or `feature-request` based on their scope
- [X] Sync issues from this repository into the new Jira instance

**Comment by kaylareopelle** [comment]
Put on hold based on formatting and issue status transition issues waiting on assistance from Workato.

**Comment by kaylareopelle** [comment]
Issue syncing complete as of 9/12/2022! 🎉

**Comment by kaylareopelle** [comment]
@angelatan2 - I wanted to let you know that the "Work has been completed" comment was printed twice on this issue.
